spec/factories/offers.rb

```diff
@@ -13,10 +13,29 @@
     created_by     {|m| m.association(:user) }
     reviewed_by_id nil
     reviewed_at    nil
+    received_at    nil
+    review_completed_at nil
 
     trait :submitted do
-      submitted_at    { Time.now }
-      state           'submitted'
+      submitted_at { Time.now }
+      state        'submitted'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.